src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -40,16 +67,33 @@ const SubscriberHistory = () => {
       <div className="dp-box-shadow col-md-8 col-sm-8" style={{ margin: '80px auto' }}>
         <div className="col-md-8 col-sm-8" style={{ padding: '30px' }}>
           Email Suscriptor
-          <div style={{ display: 'flex' }}>
-            <input type="text" value={state.subscriberSearch} onChange={handeChange} />
-            <button className="dp-button button-medium primary-grey" onClick={searchSubscriber}>
-              Buscar
-            </button>
-          </div>
+          <Formik initialValues={getFormInitialValues()} onSubmit={onSubmit}>
+            {({ submitForm, handleChange, errors, touched }) => (
+              <div>
+                <Form style={{ display: 'flex' }}>
```

Comment:
This style it's still here .... O_o

Reply:
its only for the button its not on bottom of the input, when Gus apply the styles thats with be fixed